static/app/components/createAlertButton.tsx

```diff
@@ -327,12 +324,12 @@ function CreateAlertFromViewButton({
 type Props = {
   organization: Organization;
   hideIcon?: boolean;
-  iconProps?: React.ComponentProps<typeof IconSiren>;
+  iconProps?: SVGIconProps;
   projectSlug?: string;
   referrer?: string;
   showPermissionGuide?: boolean;
 } & WithRouterProps &
-  React.ComponentProps<typeof Button>;
+  ButtonProps;
```

Comment:
I tried using only without aria label props, but the CreateAlertButton is used in a few places interchangeably so we need to respect the type.